components/FlexibleHeader/src/private/MDCStatusBarShifter.m

```diff
@@ -242,9 +260,7 @@ - (void)setEnabled:(BOOL)enabled {
 }
 
 - (BOOL)canUpdateStatusBarFrame {
-  CGRect statusBarFrame = [[UIApplication mdc_safeSharedApplication] statusBarFrame];
-  CGFloat statusBarHeight = MIN(statusBarFrame.size.width, statusBarFrame.size.height);
-  return ((statusBarHeight == kStatusBarExpectedHeight) || _statusBarReplicaView ||
+  return (![[UIApplication mdc_safeSharedApplication] isStatusBarHidden] || _statusBarReplicaView ||
```

Comment:
`isStatusBarHidden` [is deprecated](https://developer.apple.com/documentation/uikit/uiapplication/1622982-isstatusbarhidden), do we want to use it anyway?

Reply:
Bummer, that getter is really useful. Fixed.